CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ if(POLICY CMP0082)
     cmake_policy(SET CMP0082 NEW)
 endif()
 
+# The FindPythonInterp and FindPythonLibs modules are removed.
```

Comment:
Can we add version of CMake this policy was added in to the comment? e.g.
`# 3.27: The FindPythonInterp and FindPythonLibs modules are removed.`